packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -547,6 +562,23 @@ export class PluginLedgerConnectorBesu
       const out = await this.transact(txReq);
       const success = out.transactionReceipt.status;
       const data = { success, out };
+
+      // create RunTransactionV1Exchange for transaction monitoring
+      const startTimeBesuReceipt = new Date();
+      const receiptData: RunTransactionV1Exchange = {
+        request: req,
+        response: out,
+        timestamp: new Date(),
+      };
+      this.log.debug(`RunTransactionV1Exchange created ${receiptData}`);
+      this.txSubject.next(receiptData);
+
+      const endTimeBesuReceipt = new Date();
+      this.log.debug(
```

Comment:
@brunoffmateus The start and end times are going to very close to each other and will not represent the amount of time the processing of the contract invocation took at all. I'm assuming that that's what you wanted to calculate with these so I just wanted to let you know that the `txSubject.next(...)` call will return near instantly, always, because it doesn't do any processing it just passes along the message.

If this is useful info and you want to act on it then consider moving the start time to the beginning of the